core/helper_api.php

```diff
@@ -692,3 +692,30 @@ function helper_duration_to_minutes( $p_hhmm ) {
 function shutdown_functions_register() {
 	register_shutdown_function( 'email_shutdown_function' );
 }
+
+/**
+ * Generate v4 UUID
+ * @return string The v4 UUID
+ */
+function helper_generate_uuid() {
+	return sprintf('%04x%04x-%04x-%04x-%04x-%04x%04x%04x',
```

Comment:
openssl_random_pseudo_bytes might be a better/shorter solution for it (I am no specialist for this).
http://php.net/manual/en/function.openssl-random-pseudo-bytes.php
http://stackoverflow.com/questions/18830839/generating-cryptographically-secure-tokens